presto-thrift-connector-api/pom.xml

```diff
@@ -47,6 +47,19 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
 
+        <!-- for preserving javadoc and ordering of generated Thrift IDL classes, needed during compilation only -->
+        <dependency>
+            <groupId>com.facebook.swift</groupId>
+            <artifactId>swift-javadoc</artifactId>
+            <scope>provided</scope>
```

Comment:
These jars are needed for compilation only, but are not needed at runtime. "provided" scope is the one that matches it best.